content/departments/product-engineering/process/new-user-experience.md

```diff
@@ -0,0 +1,23 @@
+# Collaborating on the new user experience
+
+The new user experience (NUE) crosses the bounds of several teams' ownership areas. For example, a visitor to the product may view the home page, search several times, learn about Code Insights and then sign up for the product. Throughout this journey, they should be informed as to the major functionality and use cases of Sourcegraph as well as how to use each feature.
+
+No one team or organization can effectively own this entire experience, so careful collaboration will be required to avoid duplication of efforts and help our users find success with Sourcegraph.
```

Comment:
```suggestion
No one team or organization can effectively own this entire experience, so careful collaboration will be required to avoid duplication of efforts and help our users find success with Sourcegraph.

Major changes or more risky changes, such as adding lots of CTAs in key areas, should ideally be feature-flagged and A/B tested before being rolled out to the full fleet of customers. That way we maximize learning speed (no long-ish review process), but still do so in a way that is safe for the overall user experience.
```

Reply:
@malomarrec added your suggestion as a suggestion - looks good?

Reply:
I really don't think we should specify how any one team works in this overview, it's only about how we work together.

Reply:
@rrhyne I'm missing why this would be specific to only one team?

Reply:
Let me clarify: I don't think we should specify how the initiatives are moved forward. A/B testing is a great practice, but some projects are risky but not A/B testable, such as when there is no baseline, there is not enough traffic for the test to be significant, or the timeline makes it impossible.

Reply:
That makes sense. Maybe there's another way to convey the point here, which is that we need to make sure we think about the design/user experience, and don't create disjointed or annoying experiences.

Reply:
I support that but need to go ahead and merge this before I head out on vacation. Happy to create a PR to add this when I get back!